### PR TITLE
Feature: Satellite Endpoints Refactor

### PR DESCRIPTION
**Pull request by Umang-Rajkarnikar**
Changes made:
- Modified SatelliteModel schema to include noradId and tleLines fields
- Added global variable to store tleLines for satellites
- Added endpoint to update a satellite target's list of valid commands
- Refactored endpoints to use global tleLines variable
- Updated tests to support new changes

### CODE REVIEW

File: src/leo-server-app/src/routes/satellite.ts [hunk 1]
```diff
@@ -135,7 +125,10 @@ function isSunlit(date: Date, lon: number, lat: number, height: number) {
 }
 
 router.get("/getSatelliteInfo", (req: any, res: any) => {
+  const noradId = req.query.noradId ?? "55098";
```
Comment (quinnha) [inline src/leo-server-app/src/routes/satellite.ts:128]: can we make a default ID and then define it there

File: src/leo-server-app/src/routes/satellite.ts [hunk 2]
```diff
@@ -168,6 +164,9 @@ router.get("/getPolarPlotData", (req: any, res: any) => {
 });
 
 router.get("/getMaxElevation", (req: any, res: any) => {
+  const noradId = req.query.noradId ?? "55098";
```
Comment (quinnha) [inline src/leo-server-app/src/routes/satellite.ts:167]: same comment as above

File: src/leo-server-app/src/routes/satellite.ts [hunk 3]
```diff
@@ -196,6 +195,9 @@ router.get("/getMaxElevation", (req: any, res: any) => {
 });
 
 router.get("/getNextPasses", (req: any, res: any) => {
+  const noradId = req.query.noradId ?? "55098";
```
Comment (quinnha) [inline src/leo-server-app/src/routes/satellite.ts:198]: same as above

File: src/leo-server-app/src/routes/satellite.ts [hunk 4]
```diff
@@ -259,6 +261,9 @@ router.get("/getNextPasses", (req: any, res: any) => {
 });
 
 router.get("/getSolarIlluminationCycle", (req: any, res: any) => {
+  const noradId = req.query.noradId ?? "55098";
```
Comment (quinnha) [inline src/leo-server-app/src/routes/satellite.ts:264]: same as above

File: src/leo-server-app/src/routes/satellite.ts (hunk 1 above)
Comment (quinnha) [reply src/leo-server-app/src/routes/satellite.ts:128]: actually can we also just make this another function so it looks like const noradId = setNoradId(req.query.noradId) for readability

File: src/leo-server-app/src/tests/satellite.test.ts [hunk 5]
```diff
@@ -7,6 +7,7 @@ const {
   setTLE,
 } = require("../routes/satellite");
 
+let noradId = "55098";
```
Comment (quinnha) [inline src/leo-server-app/src/tests/satellite.test.ts:10]: same as default id global variable

**Review by quinnha — APPROVED** [review]
great pr, just one change need to make and apply it over

File: src/leo-server-app/src/routes/satellite.ts (hunk 1 above)
Comment (Umang-Rajkarnikar) [reply src/leo-server-app/src/routes/satellite.ts]: oh true, nice catch

**Pull request merged**